src/api/resource.ts

```diff
@@ -0,0 +1,86 @@
+import Axios from 'axios';
+import { Dispatch } from 'redux';
+
+import {
+  createResourceFailure,
+  createResourceSuccess,
+  getResourceFailure,
+  getResourceSuccess,
+  updateResourceFailure,
+  updateResourceSuccess,
+} from '../redux/actions/resourceActions';
+import { Resource, ResourceFileMap, ResourceMetadata } from '../types/resource.types';
+import { API_BASE_URL, StatusCode } from '../utils/constants';
```

Comment:
vi har en blanding av UPPER_CASE og CamelCase i `constants` nå , også for enums

Reply:
Godt poeng. Bør velge en. Jeg stemmer for UPPER_CASE for konstanter, og CamelCase for enums. hva synes dere @Eiriknil @placbo ?

Reply:
Enig. Uppercase for constants